catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/filter/VisitableXmlElementImpl.java

```diff
@@ -399,6 +395,43 @@ private ExpressionOrAnyElement(JAXBElement element) {
     }
   }
 
+  private static class DistanceBufferElement extends ExpressionOrAnyElement {
+
+    private double buffer;
+
+    public DistanceBufferElement(JAXBElement element) {
+      super(element);
+      if (!element.getDeclaredType().equals(DistanceBufferType.class)) {
+        throw new FilterProcessingException(
+            INVALID_INVOCATION + element.getDeclaredType().getName());
+      }
+      this.buffer = ((DistanceBufferType) element.getValue()).getDistance().getValue();
+    }
+
+    @Override
+    public List<Object> getValue() {
+      List<Object> values = super.getValue();
+      values.add(
+          new VisitableElement<Double>() {
```

Comment:
❓ Any chance this can be cleaned up? Could it be a new subclass of `VisitableXmlElementImpl` or would `LiteralElement`, which supports anything `Serializable`, be enough?

Reply:
`VisitableXmlElementImpl` and `LiteralElement` both take `JAXBElement`s in their constructors which they use to set their `value`s but this is just a wrapper around a double. I'm going to leave it as is

Reply:
Looks like this is just used to curry the `radius` value and the ctors (which take JAXB element) aren't compatible. So I'm good with leaving this as is.